lib/clockr/masterclock.ex

```diff
@@ -18,13 +18,19 @@ defmodule Clockr.Masterclock do
   """
   def start_link(clock = %{control_source_id: control_source_id}, opts \\ []) do
     {control_source_id, _} = Integer.parse(control_source_id, 16)
+    padded_control_source_id = case byte_size(:binary.encode_unsigned(control_source_id)) do
+      1 ->
+        {0, control_source_id}
+      _ ->
+        {control_source_id}
```

Comment:
@bklang Not really sure this is correct. How can I properly split the number into two distinct bytes?